capi.config.ts

```diff
@@ -26,6 +26,13 @@ export const config: CapiConfig = {
     westendDev: {
       binary: polkadot,
       chain: "westend-dev",
```

Comment:
the epoch for `westend-dev` seems to be 1 hour

<img width="183" alt="image" src="https://user-images.githubusercontent.com/1209171/231240428-4f7174cd-7389-4d3a-90fb-b6070d5eb9f8.png">


Changing the relay chain to `rococo-local` will set it to 1 minute

cc @nythrox @harrysolovay

Reply:
perhaps this is why all zombienet examples use `rococo-local`